packages/components/src/HtmlRenderer.js

```diff
@@ -45,6 +33,7 @@ exports.renderHtmlInputWithHooksImpl = function (htmlInput, hooks) {
     return htmlToReactParser.parseWithInstructions(
         htmlInput,
         isValidNode,
-        uncurriedHooks
+	uncurriedHooks,
+	uncurriedPreprocessHooks
```

Comment:
`uncurriedHooks` comes from a previous version where I was uncurrying purescript functions manually, now it's not the case anymore. Maybe we should change these names.